app/page/info/activity.tsx

```diff
@@ -17,17 +22,21 @@ const TransactionHistory: React.FC<{
   const { chainId } = useNetwork()
   const chainName = getChainName(chainId)
   return (
-    <div className="flex-col justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
-      Transaction History:
-      {txLogs.length === 0 ? (
-        <div>(No transactions)</div>
-      ) : (
-        // TODO: Render based on transaction log type
-        txLogs.map((txLog, i) => (
-          <UserOpHistoryItem key={i} txLog={txLog} chainName={chainName} />
-        ))
-      )}
-    </div>
+    <>
+      {/* Home page activity list bar */}
+      <div className="w-full flex flex-col items-start">
+        {txLogs.length === 0 ? (
+          <div></div>
+        ) : (
+          // TODO: Render based on transaction log type
+          txLogs.map((txLog, i) => {
+            return (
+              <UserOpHistoryItem key={i} txLog={txLog} chainName={chainName} />
+            )
+          })
+        )}
+      </div>
```

Comment:
how about remove the `<div/>` ?
```suggestion
        {txLogs.length > 0 &&
          // TODO: Render based on transaction log type
          txLogs.map((txLog, i) => {
            return (
              <UserOpHistoryItem key={i} txLog={txLog} chainName={chainName} />
            )
          })
        )}
      </div>
```

Reply:
Taking one step further, `txLogs` is already an array, we could get rid of `txLogs.length > 0` check and directly map on it.